alt-tab-macos/logic/Preferences.swift

```diff
@@ -79,7 +80,7 @@ class Preferences {
             tabKeyCode = try UInt16(value).orThrow()
         case "metaKey":
             let p = try metaKeyMacro.labelToMacro[value].orThrow()
-            metaKeyCode = p.preferences.0
+            metaKeyCodes = p.preferences.0.map { UInt16($0) }
```

Comment:
Just FYI I was afraid that this `map` would not guarantee to respect the input array order. It [seems it does](https://stackoverflow.com/a/31358962/2249756) :)

Reply:
In this case its not critical, it e.g. contains `[kVK_Control, kVK_RightControl]` and the usage is `let isMeta = Preferences.metaKeyCodes!.contains(event.keyCode)`

Thanks for pointing at this aspect. Good to know!